site/public/templates/grading/Mark.twig

```diff
@@ -9,7 +9,7 @@ Required Input:
     show_publish: True to show the 'publish' option
 #}
 <div id="mark-{{ mark.id }}"
-     class="mark-container row {{ mark.publish ? 'mark-publish' : '' }} {{ first_mark ? 'mark-first' : '' }} {{ not edit_marks_enabled and mark_disabled ? 'mark-disabled' : '' }} {{ mark.deleted ? 'mark-deleted' : '' }}"
+     class="mark-container key_to_click row {{ mark.publish ? 'mark-publish' : '' }} {{ first_mark ? 'mark-first' : '' }} {{ not edit_marks_enabled and mark_disabled ? 'mark-disabled' : '' }} {{ mark.deleted ? 'mark-deleted' : '' }}" tabindex="0"
```

Comment:
These sorts of headers in general don't seem to work on Firefox with the Enter key.